apps/opencs/view/render/instancemode.cpp

```diff
@@ -681,3 +697,183 @@ void CSVRender::InstanceMode::deleteSelectedInstances(bool active)
 
     getWorldspaceWidget().clearSelection (SceneUtil::Mask_EditorReference);
 }
+
+void CSVRender::InstanceMode::dropInstance(DropMode dropMode, CSVRender::Object* object, float objectHeight)
+{
+    const osg::Vec3d& point = object->getPosition().asVec3();
```

Comment:
This shouldn't be a reference. Strictly speaking the reference is okay, because it will extend the lifetime of the temporary. But nothing is gained by using a reference and it adds unnecessary complexity to the code.